src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,9 @@ public CommandResult execute() {
         }
     }
 
+	@Override
+	public boolean isMutating() {
+		return true;
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.